sections/collage.liquid

```diff
@@ -30,6 +30,7 @@
           {{ block.shopify_attributes }}
           {% if settings.animations_reveal_on_scroll %}
             data-cascade
+            style="--animation-order: {{ forloop.index }};"
```

Comment:
Adding these seems to add a lot of inline styles for a very limited effect: if a cascading section is rendered above the fold the cascading will still work before the JS takes over and handles it with the IntersectionObserver. This might happen on collection template for instance.

Personally I am OK to remove these inline styles and have those elements just fade in at the same time if they're above the fold. From a UI perspective the cascading is mostly nice when you are scrolling the page, not on initial page load. From a code perspective I feel like they add more complexity relative to the benefit.

Reply:
I don't have a strong feeling about this. On this one we should confirm with a designer if that works for them or if we keep the `animation order` 👍  

Here is the difference shown [in video](https://screenshot.click/21-15-zcxdp-avt3l.mp4). @melissaperreault what do you think ? 🤔

Reply:
Hey! I have a strong preference to keep it all the same as it felt like 2 different animations when we removed the cascade on page load.

Reply:
OK - I do have concerns about the maintainability of doing cascading two ways (one for no-JS/above-the-fold, one with JS) but if we feel strongly the design is important then this seems like the way to do it.

Reply:
I'd love to have other UX point of view though and align to find the perfect trade off! The grid in its form did gave that impression of difference that I found less aligned or consistent with the flow as you discover the next row items.

Reply:
I agree with Meli here — I think it's important to keep the cascade everywhere. 👍